README.md

```diff
@@ -1,2 +1,119 @@
-# docker-skyd
-The official docker image for the skyd daemon
+# Quick reference
+
+- **Maintained by**:  
+  [Skynet Labs](https://skynetlabs.com)
+
+- **Skyd source code and releases**:  
+  [SkynetLabs/skyd](https://gitlab.com/SkynetLabs/skyd)
+
+- **Skyd license information**:  
+  [Skynet License 1.0](https://gitlab.com/SkynetLabs/skyd/-/blob/master/LICENSE.md)
+
+- **Where to get help**:  
+  [Skynet Labs Discord](https://discord.gg/skynetlabs) or [Skynet Labs Twitter](https://twitter.com/SkynetLabs)
+
+- **Where to file issues**:  
+  [https://github.com/skynetlabs/docker-skyd/issues](https://github.com/skynetlabs/docker-skyd/issues)
+
+# Supported tags and respective `Dockerfile` links
+
+- [`1.5.5-scratch`, `1.5-scratch`, `1-scratch`, `scratch`, `latest`](https://github.com/SkynetLabs/docker-skyd/blob/main/scratch/Dockerfile)
+- [`1.5.5-bullseye-slim`, `1.5-bullseye-slim`, `1-bullseye-slim`, `bullseye-slim`](https://github.com/SkynetLabs/docker-skyd/blob/main/bullseye-slim/Dockerfile)
+
+# What is Skynet?
+
+Skynet is a decentralized storage and app hosting platform that makes it easy to join the decentralized internet movement, as a user or a web3 developer.
+
+Skynet apps transform what’s possible on the web. Beyond protecting privacy, decentralization enables application, integration, and innovation that simply cannot be replicated by the centralized world. Now, we can break free of the walled gardens and data silos that have constricted invention and interoperability. Key features of decentralization such as user-owned personal data, persistent identity across apps, and censorship-resistance will be the new standards of the digital world.
+
+> [skynetlabs.com](https://skynetlabs.com)
+
+# How to use this image
+
+See [How To Use This Image](https://github.com/skynetlabs/docker-skyd/blob/main/README.md#how-to-use-this-image) on GitHub for up-to-date documentation.
+
+### Startup command
+
+This image declares `ENTRYPOINT [ "skyd" ]` and does not override any default command line arguments. You are expected to set those yourself as a part of the command.
+
+#### Webportal stack required arguments
+
+When running skyd image as a part of the skynet webportal stack, you need to change enabled modules and make the api accessible.
+
+```yaml
+services:
+  sia:
```

Comment:
For this REPO I would change this to `skyd` because nothing in the dockerfiles is indicating sia anymore. So this will just be confusing to new users.

and on that note, it looks like we also removed the command that copied the binaries from `skyc skyd` to `siad siac`, which I think is correct for this repo.

Are we going to keep that, and finally update our portals? If so, we should update our webportal repo to say the service is skyd and we need to double check all the places that we reference siac, specifically in ansible.

Reply:
renamed in https://github.com/SkynetLabs/docker-skyd/pull/1/commits/7a9ab217e5d983a6f229b58a5ff8cd309917be75